### ISSUE TITLE
ICS for Fabric IBC

### ISSUE THREAD

**Issue by hxrts** [datachainlab/public-docs#8]
Just found your Fabric work and it looks great! I'm wondering if you'd be interested in adding some of this documentation to the [ICS spec](https://github.com/cosmos/ics/tree/master/spec) for better visibility. If so I can have the IBC core team take a look and give feedback.

I also just spoke with the [IRIS team](https://github.com/irismod), who wanted to contribute to a Fabric-IBC implementation and might be interested in collaborating. Happy to put you in touch if you like.

**Comment by storiumi** [comment]
Sam, thanks for reaching out!
I am Susumu, business development manager at Datachain.
For the ICS spec you mentioned, we are very much eager to working on the documentation of IBC-fabric. It would be really great if we could ask your advice for preparing docs.
We are also very happy to look for opportunities for collaboration with IRIS team. Jun (CTO) ,Ryo (engineer) and I are people in charge.
If there is forum or so, please invite us. We could also set up a call if it suits you. Let us know how to communicate with the team
- Jun(@bluele) jun.kimura@
- Ryo(@3100) ryo.sato@
- Susumu(@storiumi) susumu.toriumi@
Company mail address is "name"@datachain.jp

**Comment by hxrts** [comment]
Yes, that would be great. Sending an email to your team + IRIS.

**Comment by harrietcao** [comment]
Hello Susumu and Datachain team, looking forward to the collaboration! Would love to get a call together, to hear about your plan and insights. We also have some potential use cases of fabric IBC  and would love to exchange

**Comment by storiumi** [comment]
Hi Harriet! Thanks for reaching out.
Let us set up the call then. I assume IRIS team is Hong-kong base(GMT +8)
Let me look for the available slot (probably) next week that suits you.

**Comment by storiumi** [comment]
Hi Harriet @harrietcao , sorry for kept you waited.
We would like to set up a call (1hr) on 15th Jan, 14:30 or later that day (GMT+8)
It would be nice if you could confirm the above schedule suit you. 
Also shall I invite you on the address with @bianjie.ai? Please let us know.

**Comment by harrietcao** [comment]
Yes, sure, my email address starts as harriet. I sent you an intro email last weekend and might get to your trash folder. :)  Thanks a lot for setting the call. Looking forward to the discussion!

**Comment by storiumi** [comment]
Thanks! Just sent you invitation. 
Please check your schedule as we set our call from 14:00 on 15th 🙇
